app/controllers/accept_account_invitations_controller.rb

```diff
@@ -4,8 +4,13 @@ class AcceptAccountInvitationsController < ApplicationController
   def show
     return if user_signed_in?
 
-    session[:after_sign_in_url] = request.fullpath
-    redirect_to new_user_session_url(email: AccountInvitation.find_by!(token: ps.fetch(:token)).email)
+    session[:after_authentication_url] = request.fullpath
```

Comment:
Why changed the key `:after_sign_in_url` to `:after_authentication_url`? `:after_authentication_url` is longer,  harder to write and spell.